docs/README.md

```diff
@@ -0,0 +1,12 @@
+## AICOE - SRE Docs
+
+Here you will find a compilation of documentation that help achieve efficiency, transparency, and uniformity of performance. 
```

Comment:
What does this actually mean? :sweat_smile:

Reply:
this is ripped from the definition of an SOP